migrate/migrate.go

```diff
@@ -17,6 +18,8 @@ import (
 	pipep "github.com/heetch/migrate/pipe"
 )
 
+var nowFunc = time.Now
```

Comment:
You only need the current time to generate a timestamp, so if your function directly returns a timestamp directly you can simplify both your application and test code